.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
Consider using environment variable management libraries or .env files to keep sensitive keys like GITHUB_TOKEN and OPENAI_API_KEY secure and avoid hard-coding them directly in the script.